packages/x-date-pickers-pro/src/MobileNextDateRangePicker/MobileNextDateRangePicker.tsx

```diff
@@ -50,6 +49,10 @@ const MobileNextDateRangePicker = React.forwardRef(function MobileNextDateRangeP
         sx,
         ref,
       }),
+      toolbar: {
+        hidden: false,
```

Comment:
Do you think it's worth adding the `false` value ? If we just remove those lines, the toolbar receives `undefined` which works as well.

Reply:
Yes, it does work with undefined, but I usually like being explicit. 🤔 
We can omit it if you think it's pointless. But on the hand—if we will ever need to change the default rule, then it will be less changes and a bit easier to discover places, where it need to happen. 🤔

Reply:
True
Let's be explicit, I'll also add it to my `tabs` PR.